### PR TITLE
(copy changes): Change the search order place holder to sync with the mocks.[03-1714]

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary
- Changing the placeholder wording in the drug search.
- Removing the encounterProvider array which is not really required by the BE and was causing a bug when the logged in user is not a provider.

## Screenshots
Screenshot 2022-12-22 at 18 25 46

## Related Issue
https://issues.openmrs.org/browse/O3-1714

### CODE REVIEW

File: packages/esm-patient-medications-app/src/api/api.ts [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-import {
-  FetchResponse,
-  openmrsFetch,
-  Session,
-  useConfig,
-  OpenmrsResource,
-  showNotification,
-  useSession,
-} from '@openmrs/esm-framework';
+import { FetchResponse, openmrsFetch, Session, useConfig, OpenmrsResource, useSession } from '@openmrs/esm-framework';
```
Comment (hadijahkyampeire) [inline packages/esm-patient-medications-app/src/api/api.ts:3]: Just some clean up i.e removing unused import

**Comment by vasharma05** [comment]
Hi @hadijahkyampeire , I remember there were more changes to the search UI other than the placeholders

**Comment by hadijahkyampeire** [comment]
The other one was the clear button, those are the visual changes but I talked to you about the clear word, I think it is not supported by carbon. And the other tasks are actually more of logic changes which need more time and their own PR to address them, I usually try to avoid having unrelated problems in one PR, I find it big for the reviewer and it usually takes longer to get it merged.

these two issues will be in another PR the rest are addressed
- [ ]  The order basket remains open when the patient is switched
- [ ]  The order basket doesn’t close when switching a patient when there are pending orders.

**Comment by vasharma05** [comment]
Thanks for the work @hadijahkyampeire!

**Review by vasharma05 — APPROVED** [review]
Looks great!
Thanks!

**Pull request merged**